validphys2/src/validphys/coredata.py

```diff
@@ -393,28 +393,37 @@ def export_data(self, buffer):
         ret = {"data_central": self.central_values.tolist()}
         yaml.safe_dump(ret, buffer)
 
+    def sort_definitions_by_treatment(definitions, orderkey):
+        sorted_definitions={}
+        for orderkey in orderkey:
+            for key in definitions:
+                if definitions[key]["treatment"] == orderkey:
+                    sorted_definitions[key] = definitions[key]
+        return sorted_definitions
+
     def export_uncertainties(self, buffer):
         """Exports the uncertainties defined by this commondata instance to the given buffer"""
         definitions = {}
         for idx, row in self.systype_table.iterrows():
-            definitions[f"sys_{idx}"] = {"treatment": row["treatment"], "type": row["name"]}
-
+            if row["name"] != "SKIP":
+                definitions[f"sys_{idx}"] = {"treatment": row["treatment"], "type": row["name"]}
+        orderkey = ["ADD", "MULT"]
+        sorted_definitions = sort_definitions_by_treatment(definitions, orderkey)
```

Comment:
```suggestion
        # Order the definitions by treatment as ADD, MULT
        # TODO: make it so that it corresponds to the original order exactly
        sorted_definitions = {k: v for k, v in sorted(definitions.items(), key=lambda item: item[1]["treatment"])}
```

Since this is just going to be a temporary solution, let's make it clear that it is so.